tools/cpat/cpat.xml

```diff
@@ -144,6 +166,41 @@
             </output>
             <output name="no_orf_seqs" file="test02_no_orgs.txt" ftype="txt"/>
         </test>
+        <test expect_num_outputs="4">
+            <param name="gene" value="sequences.fasta.gz"/>
+            <conditional name="ref_source">
+                <param name="source" value="builtin" />
+                <param name="ref_fasta" value="test1.fasta.gz" />
```

Comment:
ref_fasta_builtin I assume

Reply:
I have defined the value as `builtin` as below

```
<param type="select" label="Reference genome source" name="source">
    <option value="history" selected="true">Use from History</option>
    <option value="builtin">Use Built-in</option>
</param>
```

Reply:
But ref_fasta is not a Param in the built-in case isn't it? On my mobile, no maybe I don't see it

Reply:
Yes I checked it, it should be ref_fasta_builtin I have made the change in recent commit